### PR TITLE
Document X-Audit-Log-Reason on webhook endpoints

### PR DESCRIPTION
**Pull request by lts20050703**
Closes: #1906

### CODE REVIEW

File: src/types/webhooks/createWebhook.ts [hunk 1]
```diff
@@ -3,4 +3,5 @@ export interface CreateWebhook {
   name: string;
   /** Image for the default webhook avatar */
   avatar?: string | null;
+  reason?: string;
```
Comment (Skillz4Killz) [inline src/types/webhooks/createWebhook.ts:6]: add a /** */ also what about the function itself?

File: src/types/webhooks/modifyWebhook.ts [hunk 2]
```diff
@@ -6,4 +6,5 @@ export interface ModifyWebhook {
   avatar?: string | null;
   /** The new channel id this webhook should be moved to */
   channelId?: string;
+  reason?: string;
```
Comment (Skillz4Killz) [inline src/types/webhooks/modifyWebhook.ts:9]: add a /** */ also what about the function itself?

**Review by Skillz4Killz — CHANGES_REQUESTED** [review]

File: src/types/webhooks/createWebhook.ts (hunk 1 above)
Comment (lts20050703) [reply src/types/webhooks/createWebhook.ts:6]: the function should work?
image

File: src/types/webhooks/modifyWebhook.ts (hunk 2 above)
Comment (lts20050703) [reply src/types/webhooks/modifyWebhook.ts:9]: The function should work?
image

**Review by Skillz4Killz — APPROVED** [review]

**Pull request closed without merging**